### ISSUE TITLE
documentation is missing half of the side bar

### ISSUE THREAD

**Issue by typemytype** [typemytype/drawbot#205]
I guess this has todo with a sphinx update

left sphinx 1.7.4 (online), right sphinx 1.5.5 (local)

<img width="869" alt="screen shot 2018-05-22 at 09 23 10" src="https://user-images.githubusercontent.com/1190358/40347784-c251b23c-5da1-11e8-9522-f7c3cf3c3182.png">

**Comment by typemytype** [comment]
quick dirty fix here https://github.com/typemytype/drawbot/commit/4b8c561099081606ceb03f666e0e72fb8af96b29

the docs needs an upgrade to py3 and newer sphinx version (for later)

**Comment by thundernixon** [comment]
I haven't completely solved this, but I'm about halfway there.

In the Sphinx-Doc docs for [templating](http://www.sphinx-doc.org/en/1.4.9/templating.html), I've found:

> `sidebartoc`
>  The table of contents within the sidebar.
>
> _Deprecated since version 1.0._

So, we need to remove this in `index.html`:

```
{% block sidebartoc %}
    {{ toctree(includehidden=True) }}
{% endblock %}
```

...and (I think) place the `{{ toctree(includehidden=True) }}` into `sidebar1`.

I've made a branch, and I'm still working to get right mixture of code to make it work how it previously did.

**Comment by thundernixon** [comment]
Well, I got part of the way there.

For some reason, I'm finding it difficult to:
- Get the next/prev links in the left-hand sidebar
- Not display the "Table of Contents" of the index page (this just links to each header, which is pointless)

I think that the pagination and ToC display on the right-hand side are coming from the `relbar1` block, but I'm baffled why they're being inserted after the main page content (and thus showing up on the right-hand side) because **a)** `relbar1` is supposed to act like `sidebar1` and insert itself before the main content, and **b)** I've inserted the `relbar1` block inside the html for the sidebar (and after the sidebar block, and before ... each way has its own issues).

![image](https://user-images.githubusercontent.com/7355414/46156889-8800fc80-c248-11e8-8ad2-42c23f73720b.png)

---

If I take out `relbar1` and `relbar2` altogether, I weirdly get a "Navigation" section added, either at the top, bottom, or both:

![image](https://user-images.githubusercontent.com/7355414/46157278-5a688300-c249-11e8-9c7e-82c965b5e301.png)

The only way to avoid this unexpected addition is to include both the `relbar1` above the sidebar block and `relbar2` below the document block.

Sphinx is troubling. :|
